ui/app/mainui/AppMain.qml

```diff
@@ -1647,16 +1654,25 @@ Item {
 
                 onClosed: {
                     sendModal.closed()
+                    sendModal.modalHeaderText = ""
+                    sendModal.interactive = true
                     sendModal.preSelectedSendType = Constants.SendType.Unknown
                     sendModal.preSelectedHoldingID = ""
                     sendModal.preSelectedHoldingType = Constants.TokenType.Unknown
                     sendModal.preSelectedAccountAddress = ""
                     sendModal.preSelectedRecipient = undefined
                     sendModal.preDefinedAmountToSend = ""
                     sendModal.preSelectedChainId = 0
+
+                    sendModal.stickersPackId = ""
+                    sendModal.publicKey = ""
+                    sendModal.ensName = ""
                 }
             }
             onLoaded: {
+                if (!!sendModal.modalHeaderText) {
```

Comment:
Those `if`s here seem to be not necessary. Assigning the same, empty value is not a problem here.